Tests/PackageGraphPerformanceTests/DependencyResolverPerfTests.swift

```diff
@@ -174,14 +174,17 @@ extension ProductFilter: JSONSerializable, JSONMappable {
         switch self {
         case .everything:
             return "all".toJSON()
-        case .specific(let products):
+        case .specific(var products, let includeCommands):
```

Comment:
I patched this up to do something sane. It does not match the actual files any more than `main` does, but will be operable if someone makes a new file.